locale/loris.pot

```diff
@@ -168,14 +162,3 @@ msgstr ""
 
 msgid "Download Table as CSV"
 msgstr ""
```

Comment:
Idem here, this translations were merged and belong to `loris namespace` , why we are deleting the in this PR ?